src/jit/lclvars.cpp

```diff
@@ -2689,6 +2689,10 @@ void Compiler::lvaSortByRefCount()
     lvaTrackedCount             = 0;
     lvaTrackedCountInSizeTUnits = 0;
 
+#ifdef DEBUG
+    VarSetOps::AssignNoCopy(this, lvaTrackedVars, VarSetOps::MakeEmpty(this));
```

Comment:
On second thought, would it better to add a null check in lsra before trying to use lvaTrackedVars rather than initializing it to empty set in debug builds?  Because, if some other code path uses lvaTrackedVars like LSRA did, then it will go unnoticed on debug builds.  Since JitStress/LSRA stress run is not done on retail bits, the issue will not be caught unless we have a directed test, which we don't have otherwise that test would have caught it instead of stress mode.